ludicrousdb/includes/class-ludicrousdb.php

```diff
@@ -489,16 +499,17 @@ public function db_connect( $query = '' ) {
 			return $this->dbh;
 		}
 
-		// Determine whether the query must be sent to the master (a writable server)
-		if ( ! empty( $use_master ) || ( $this->srtm === true ) || isset( $this->srtm[ $this->table ] ) ) {
+		$is_write = $this->is_write_query( $query )
+		// Determine whether the query must be sent to the master (a writable server).
+		if ( ! empty( $use_master ) || ( true === $this->srtm ) || isset( $this->srtm[ $this->table ] ) ) {
 			$use_master = true;
-		} elseif ( $is_write = $this->is_write_query( $query ) ) {
+		} elseif ( $is_write ) {
 			$use_master = true;
 			if ( is_array( $this->srtm ) ) {
 				$this->srtm[ $this->table ] = true;
 			}
 
-		// Detect queries that have a join in the srtm array.
+			// Detect queries that have a join in the srtm array.
```

Comment:
This is incorrect.

Inline comments that are above the line they are commenting on should be indented equally.